modules/metastore/src/Storage/MetastoreEntityStorageInterface.php

```diff
@@ -7,28 +7,4 @@
  */
 interface MetastoreEntityStorageInterface extends MetastoreStorageInterface {
```

Comment:
Can this file be deleted now? It doesn't look like the interface is being used anywhere else.

Reply:
Maybe. Probably. This was one of several things I considered doing in this PR but decided to hold back on to minimize the disruption in one PR. I think when we're all back on this full time we should have a less rushed conversation about further refactor/consolidation of the classes and interfaces in the metastore module, but I'd like to hold off on deleting this completely until we can do that.